### DISCUSSION TITLE
cgapp command not found

### DISCUSSION THREAD

**Discussion by JOduMonT** [create-go-app/cli#122] [General]
- go 1.17.8 is installed and works
- cgapp@latest has been installed with go
- cgapp command not found
- neither: `$GOPATH/bin/cgapp create` == `bash: /bin/cgapp: No such file or directory`
OS: Ubuntu 20.04LTS
![image](https://user-images.githubusercontent.com/5204724/158925397-06ca532d-9069-4f21-8d6b-e7500e2eba46.png)

**Comment by koddr** [comment]
Hi,

Please check your `$GOPATH` (where Go install packages on your system). 
Furthermore, check the path to `cgapp` binary, like this: `whereis cgapp`.

**Reply by JOduMonT** [reply]
> Please check your `$GOPATH` (where Go install packages on your system).

````bash
whereis go
go: /usr/local/go /usr/local/go/bin/go
````

echo $GOPATH goes nowhere :p aka it returns nothing.

But in the means while I found: `$HOME/go/bin/cgapp`

my question now is, could I move that go/bin/* somewhere like /usr/local/bin/

**Reply by koddr** [reply]
Yes, you can attempt to move the binary to any other directory and then write an alias for it.
